finance.reckoner.Reckoner.yaml

```diff
@@ -0,0 +1,39 @@
+app-id: finance.reckoner.Reckoner
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: reckoner
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=home
```

Comment:
The default backup location seems to be ~/Documents. Does this need to be home or can xdg-documents work?